torchgeo/datasets/nccm.py

```diff
@@ -0,0 +1,244 @@
+"""Northeastern China Crop Map Dataset."""
+
+import glob
+import os
+import shutil
+from collections.abc import Iterable
+from typing import Any, Callable, Optional, Union
+
+import matplotlib.pyplot as plt
+import torch
+from matplotlib.figure import Figure
+from rasterio.crs import CRS
+
+from .geo import RasterDataset
+from .utils import BoundingBox, download_url, extract_archive
+
+
+class NCCM(RasterDataset):
```

Comment:
Does anyone else refer to the dataset as NCCM or just us? Want to make sure this abbreviation is correct. Could also be "Crop Type Map".

Reply:
this was a naming scheme that we decided. NCCM stands for Northeastern China Crop Map. is that fine?